Guides/organising.md

```diff
@@ -58,10 +64,10 @@ Contributions can be lots of different things, and so we use the [emoji key](/RE
 
 We welcome any administrative and organisational help you can offer! Please get in touch via [email](mailto:xgov-data-ethics@proton.me). We'd especially like to hear from people who are new to the Civil Service and from groups usually under-represented in data science.
 
-### I'd like to lead a session on a particular topic
+### I'd like to lead a session on a particular topic or book
 
 Brilliant! Please reach out to us via [email](mailto:xgov-data-ethics@proton.me) and we can set this up with you.
 
 ## Communicating
 
-We primarily use our [newsletter](https://xgovdataethics.substack.com/) to communicate with interested Data Science practitioners across government. Sign up to hear about upcoming events.
+We primarily use our Substack [newsletter](https://xgovdataethics.substack.com/) to communicate with interested Data Science practitioners across government. Sign up to hear about upcoming events.
```

Comment:
Higher level to avoid tooling content? Can link to newsletter section of tooling page?

Reply:
think we should remove data science here as we are now more wide reaching